Gemfile

```diff
@@ -58,14 +58,17 @@ gem 'devise_invitable', '~> 2.0.0'
 gem 'cancancan'
 gem 'webpacker'
 
+group :demo, :development, :test do
+  gem 'factory_bot_rails'
+end
```

Comment:
FactoryBot is really helpful for this, and we would need it to be available on demo